packages/web3-core/package.json

```diff
@@ -2,7 +2,15 @@
 	"name": "web3-core",
 	"version": "4.0.1-rc.0",
 	"description": "Web3 core tools for sub-packages. This is an internal package.",
-	"main": "lib/index.js",
+	"main": "./lib/commonjs/index.js",
+	"module": "./lib/esm/index.js",
+	"exports": {
+		".": {
+			"import": "./lib/esm/index.js",
+			"require": "./lib/commonjs/index.js",
+			"types": "./lib/types/index.d.ts"
```

Comment:
`types` must always be first in list and `default` last: https://nodejs.org/api/packages.html#community-conditions-definitions

Reply:
Thanks